src/Microsoft.ML.CodeGenerator/CodeGenerator/CSharp/TransformGeneratorFactory.cs

```diff
@@ -14,6 +14,15 @@ internal interface ITransformGenerator
         string[] GenerateUsings();
     }
 
+    // For orphan Transformer ( transformer that doesn't exist in AutoML )
```

Comment:
Why not add them in AutoML?

Reply:
Because AutoML doesn't have the corresponding transformer to that, Those orphan transformer are a working around way by model builder, which uses these transformers for creating pipeline (just pipeline) only for azure image. And I'd like to keep AutoML from those hacking

Reply:
Ok. Sounds good.